src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrderingPartyController.cs

```diff
@@ -74,18 +76,70 @@ public async Task<ActionResult<OrderingPartyModel>> GetAsync(string orderId)
         }
 
         [HttpPut]
-        public ActionResult Update(string orderId, OrderingPartyModel model)
+        public async Task<ActionResult> Update(string orderId, OrderingPartyModel model)

```

Comment:
Can the method name be now changed to UpdateAsync, as you are now using async Task